<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathParserMainWindow</class>
 <widget class="QMainWindow" name="MathParserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MathParser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="formulaLabel">
      <property name="text">
       <string>Formula</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="formulaLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="variablesLabel">
      <property name="text">
       <string>Variables</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="variablesComboBox"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="variablesLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="resultLineEdit"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="calculatePushButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QGraphicsView" name="plotGraphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
